src/condor_utils/file_transfer.cpp

```diff
@@ -256,6 +257,17 @@ class FileTransferItem {
 	filesize_t m_file_size{0};
 };
 
+void
+dPrintFileTransferList( int flags, const FileTransferList & list, const std::string & header ) {
+	std::string message = header;
+	for( auto & i: list ) {
+		formatstr_cat( message, " %s -> %s [%s],",
+			i.srcName().c_str(), i.destDir().c_str(), i.destUrl().c_str()
+		);
+	}
+	dprintf( flags, "%s\n", message.c_str() );
```

Comment:
Would be nice to not have a trailing comma at the end of the log message.

Reply:
Done.